docs/pages/versions/unversioned/sdk/maps.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Maps
+sidebar_title: Maps (expo-maps)
+description: A library that provides access to Google Maps on Android and Apple Maps on iOS.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-maps'
+packageName: 'expo-maps'
+platforms: ['ios', 'android']
+isNew: true
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Step } from '~/ui/components/Step';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+> **warning** **Expo Maps is currently in alpha and subject to breaking changes. It is not available in the Expo Go app.**
+
+## Installation
+
+<APIInstallSection />
+
+## Configuration
+
+Expo Maps provides access to the platform native map APIs on Android and iOS.
+
+- **Apple Maps is available on iOS only**. No additional configuration is required to use it after installing this package.
+- **Google Maps is available on Android only**. While Google provides a Google Maps SDK fore iOS, Expo Maps supports it exclusively on Android. If you want to use Google Maps on iOS, you can look into a using an [alternative library](https://reactnative.directory/) or [writing your own](/modules/overview/).
+
+**Before you can use Google Maps on Android**, you need to register a Google Cloud API project, enable the Maps SDK for Android, and add the associated configuration to your Expo project.
+
+<Collapsible summary="Set up Google Maps on Android">
+
+> If you have already registered a project for another Google service on Android, such as Google Sign In, you enable the **Maps SDK for Android** on your project and jump to step 4.
+
+<Step label="1">
+**Register a Google Cloud API project and enable the Maps SDK for Android**
```

Comment:
copied these instructions over from our [mapview](https://docs.expo.dev/versions/latest/sdk/map-view/) docs, adjusted slightly to make more sense here